front/components/items/Appointment/helper.ts

```diff
@@ -1,21 +1,21 @@
-import { strapiAppointmentQuery } from 'lib/strapi/appointments/queryType';
+import { propsAppointment, propsOfFormingDate, propsOfGetId } from './models';
 
-export type filters = {
-    [Keys: string]: Object;
-};
-type props = {
-    appointments: strapiAppointmentQuery[] | undefined;
+export const formingDate = ({ chosenDate }: propsOfFormingDate) => {
```

Comment:
```suggestion
export const formingDate = ({ chosenDate }: propsOfFormingDate) => new Date(chosenDate?.getTime() - 60 * 1000 * -120).toISOString().slice(0, 10)
```
I think it'll be more readable

Reply:
moreover I'd rename function. Maybe formatDate